docs/competition/teams/process-docs/sr2019-orientation-and-pit-rules.md

```diff
@@ -0,0 +1,81 @@
+
+[http://srobo.org/comp/league](https://studentrobotics.org/comp/league) The current league scores.
+
+[http://srobo.org/comp/team](https://studentrobotics.org/comp/team) Statistics about your team and others.
+
+## Schedule
+
+**Saturday 1/4/2017 (Day 1)**
+
+| Time | Activity |
+| ---- | -------- |
+| 9:00 | Doors open |
+| 9:30 | Introduction and Safety briefing: **All team members and team leaders must attend this briefing which will be held at arena A** |
+| 9:45 | Tinkering time |
+| 11:15 | League matches |
+| 12:15 | Lunch and Tinkering time |
+| 13:00 | League matches |
+| 17:30 | End of day |
+
+**Sunday 2/4/2017 (Day 2)**
+
+| Time | Activity |
+| ---- | -------- |
+| 9:00 | Doors open |
+| 9:30 | League matches |
+| 12:15 | Lunch |
+| 13:15 | League matches |
+| 14:30 | Competition Photo: Everyone will be gathered together with their robots for a photo. |
+| 14:45 | Knockout matches |
+| 17:00 | End of Knockout matches |
+| 17:15 | Prize Ceremony |
+| 18:00 | End of day |
+
+
+## Tinkering Time
+
+During parts of the competition scheduled as “Tinkering time”, the arenas will be open for robots to be tested in the competition environment. **You must book a slot in an arena in advance.** Approach the competition staging area (next to the arenas, see map) up to half an hour before and during sessions to request a slot.
+
+## How the competition runs
+
+Student Robotics features a series of league matches, then knockout rounds seeded by league scores. The schedule (on the website; see above link) defines when league matches will be for your team. You are responsible for bringing your robot to the arena, and taking part in the match:
+
+1.	You **must** arrive at the competition staging area (next to the arenas, see map) with your robot at least 2 minutes before your match starts.
+2.	Only your designated “robot wrangler” may enter the staging area.  They **must** be wearing a high vis jacket identifying your team.  You will receive this jacket once your robot has been checked to be compliant with the regulations in the rules.
+3.	In the staging area, a special USB flash disk will be present to tell the robot which corner it starts in. You will need to insert this into your robot **before** powering on.
+4.	Robot wranglers should turn your robot on **before** it enters the arena, so that it is ready to start on time.  The match will begin regardless of whether your robot is ready.
+5.	Robots are started by the team member pressing the start button on the power board. Once pressed, the team member should step back to the edge of the crowd barrier.
+6.	You may only enter the arena when given permission to do so by a match scoring official.
+7.	If you wish to resign from a match whilst it is in play, you may signal to a match scoring official that you wish to “throw in the towel”.  They will then turn off your robot and you will forfeit that match.
```

Comment:
I don't believe we are allowing reams to resign

Reply:
Considering there's a strategic element to where the robot stops, this sounds like a fair restriction. Deferring final vote to @prophile

Reply:
Disregarding as all orientation information is being stripped out